velox/exec/tests/TableScanTest.cpp

```diff
@@ -3033,3 +3033,25 @@ TEST_F(TableScanTest, readMissingFieldsWithMoreColumns) {
     ASSERT_EQ(stringCol->valueAt(j), fruitViews[j % fruitViews.size()]);
   }
 }
+
+TEST_F(TableScanTest, varbinaryToVarchar) {
```

Comment:
This test verifies that table scan with varbinary partition key, right? Let's rename to varbinaryPartitionKey

Reply:
Thanks, I Updated it.